db/migrate/20230510113501_create_customers_tax_rates.rb

```diff
@@ -2,7 +2,7 @@
 
 class CreateCustomersTaxRates < ActiveRecord::Migration[7.0]
   def change
-    add_column :tax_rates, :applied_by_default, :boolean, null: false, default: false
+    add_column :tax_rates, :applied_to_organization, :boolean, null: false, default: false
```

Comment:
I'm not sure you want to touch this migration :)

Reply:
thanks :)